cmd/collector/main.go

```diff
@@ -23,14 +23,13 @@ func main() {
 		log.Fatalf("failed to build default components: %v", err)
 	}
 
-	info := component.ApplicationStartInfo{
-		ExeName:  "collector",
-		LongName: "Hypertrace Collector",
-		Version:  Version,
-		GitHash:  GitHash,
+	info := component.BuildInfo{
+		Command:     "hypertrace-collector",
```

Comment:
this should still be collector right?  

If you do want to change the name of the executable, we'll need to update the Dockerfile too.